src/manage.py

```diff
@@ -217,8 +217,7 @@ def update_registry_info(self):
     # Check if there's a NPM scope or the default scope @@npm.
     assert self.scope.startswith('@')
 
-    headers = {'Accept': 'application/vnd.npm.install-v1+json; q=1.0, application/json; q=0.8, */*'}
-    response = urlfetch.fetch(util.npm_registry_url(self.scope, self.package), headers=headers, validate_certificate=True)
+    response = util.registry_get(self.scope, self.package)
     try:
       package = json.loads(response.content)
```

Comment:
Hmm, those lines aren't really needed. I think we want to assume that we are getting a JSON response and fail loudly if the registry isn't complying with that.